src/Components/Facility/ConsultationForm.tsx

```diff
@@ -478,6 +478,19 @@ export const ConsultationForm = (props: any) => {
           }
           return;
         }
+
+        case "verified_by":
+          if (
+            !state.form[field] ||
+            state.form[field] === " " ||
+            !state.form[field].replace(/\s/g, "").length
```

Comment:
@skks1212 do we need the 1st and 2nd condition, shouldn't this be covered in the 3rd condition?
```suggestion
            !state.form[field].replace(/\s/g, "").length
```